test/index.test.js

```diff
@@ -130,7 +130,7 @@ describe('Given invalid input for ', () => {
 });
 
 describe('Catching errors in updateVpcConfig ', () => {
-  it('AWS api call describeVpcs fails', () => {
+  it('AWS api call describeVpcs fails', (done) => {
```

Comment:
You shouldn't need to do this. Returning the promise should be sufficient. Try checking that travis is running the right version of node/mocha